pkgs/development/python-modules/hbmqtt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, transitions, websockets, passlib, docopt, pyyaml, nose }:
+
+buildPythonPackage rec {
```

Comment:
Shouldn't you have a `disabled = !isPy3k` since as you mentioned this only works on python3+?

Reply:
Yes, but `websockets` is py3 only as well, so it's automatic. I'll add it anyway.